sys/shell/commands/sc_netif.c

```diff
@@ -267,6 +304,37 @@ static void _netif_set_addr(kernel_pid_t dev, ng_netconf_opt_t opt,
     puts("");
 }
 
+static void _netif_set_state(kernel_pid_t dev, char *state_str)
+{
+    ng_netconf_state_t state;
+    if (strcmp("off", state_str) == 0) {
```

Comment:
Though I'm not sure about the portability of `<strings.h>`… it's defined in the POSIX module, but it get's kind of hairy if they are there (as e.g. in newlib or on native). Probably better just to expect either upper-case here as well or output lower-case in `_netif_get_state()`